src/pages/Landing.tsx

```diff
@@ -9,27 +9,34 @@ import {
     ScrollButton,
     LoadingAnimation,
 } from '@components';
-import { HeroAboutDesktop } from '@assets';
+import { HeroAboutDesktop, HeroAboutMobile } from '@assets';
 
 const Landing: React.FC = () => {
     const [isLoading, setIsLoading] = useState(true);
-    const [componentLoaded, setComponentLoaded] = useState(false);
+    const [desktopImageLoaded, setDesktopImageLoaded] = useState(false);
+    const [mobileImageLoaded, setMobileImageLoaded] = useState(false);
     const [timerFinished, setTimerFinished] = useState(false);
 
     useEffect(() => {
+        // Load Desktop SVG
         const desktopImage = new Image();
         desktopImage.src = HeroAboutDesktop;
-        desktopImage.onload = () => setComponentLoaded(true);
+        desktopImage.onload = () => setDesktopImageLoaded(true);
 
+        // Load Mobile SVG
+        const mobileImage = new Image();
+        mobileImage.src = HeroAboutMobile;
+        mobileImage.onload = () => setMobileImageLoaded(true);
+
+        // Timer
         setTimeout(() => setTimerFinished(true), 2000);
     }, []);
 
-    // Once the timer and component are finished -> display the page
     useEffect(() => {
-        if (componentLoaded && timerFinished) {
+        if (desktopImageLoaded && mobileImageLoaded && timerFinished) {
             setIsLoading(false);
         }
-    }, [componentLoaded, timerFinished]);
+    }, [desktopImageLoaded, mobileImageLoaded, timerFinished]);
 
     if (isLoading) {
```

Comment:
```suggestion
    const isLoading = !(desktopImageLoaded && mobileImageLoaded && timerFinished);
    if (isLoading) {
```

This can reduce the times React will re-render, each onload will cause a re-render, and then the `setIsLoading` in the effect will cause another one.